commands/cli.go

```diff
@@ -64,7 +68,21 @@ func parseDurationParam(paramValue string) (*time.Time, error) {
 }
 
 func awsNuke(c *cli.Context) error {
+	specificRegion := c.StringSlice("region")
```

Comment:
Wait, so you want to specify multiple regions or one region?

Reply:
specify one or multiple regions `--region eu-west-3 --region eu-west-1` here I want to nuke the resources in eu-west-1 and eu-west-3 instead of going through all regions.

Reply:
That makes sense. Could you update the README to mention this?

Reply:
Also, perhaps rename the variable from `specificRegion` to something like `selectedRegions`?